api-auto-registration/installation.hbs.md

```diff
@@ -79,10 +71,12 @@ To install API Auto Registration:
     kubectl get secret tap-values -n tap-install -o jsonpath="{.data['tap-values\.yaml']}" | base64 -d | yq '.tap_gui.app_config.app.baseUrl'
     ``` 
 
-1. (Optional) Create `api-auto-registration-values.yaml`
+1. (Optional but recommended) Create `api-auto-registration-values.yaml`
 
     If you'd like to overwrite the default values when installing the package, create a `api-auto-registration-values.yaml` file that looks like this:
 
+    This `cluster_name` field will be used as the lifecycle attribute for API entities created by the controller in TAP GUI.
```

Comment:
This line seems to be out of place. It's in between "file that looks like this" and the actual sample file. Aside from that, the description of this field is listed above with the sample `valuesSchema` output, so maybe we don't need to repeat it?

Reply:
You are right, I will remove it